examples/console_log/src/lib.rs

```diff
@@ -22,7 +18,4 @@ pub fn run() {
     log("Hello from Rust!");
     log_u32(42);
     log_many("Logging", "many values!");
-
-    console::log(JsValue::from("Another message from rust!"));
```

Comment:
Again: why?

Reply:
This one is because variadic support has been removed (as it was erroneously there before!)

Reply:
Yes, I have accidentally added variadic support while unifying how single and variadic arguments work, but I thee decided to keep it like this because many operations with variadic arguments are useful even if you actually can call them using only a single argument.

Reply:
It's true yeah a single argument works, but I think we want to discuss this a bit to figure out what future support for these APIs will look like and avoid having our hands tied by having an existing convention